programs/README.md

```diff
@@ -13,7 +13,7 @@ Folder | Name  | Description | Category | Status | Help
 ---|---|---|---|---|---
 [witness_node](witness_node) | Witness Node | Main software used to sign blocks or provide services. | Node | Active | `./witness_node --help`
 [cli_wallet](cli_wallet) | CLI Wallet | Software to interact with the blockchain by command line.  | Wallet | Active | `./cli_wallet --help` 
-[delayed_node](delayed_node) | Delayed Node | Runs a node with `delayed_node` plugin loaded. This is deprecated in favour of `./witness_node --plugins "delayed_node"`. | Node | Deprecated | `./delayed_node --help`
+delayed_node | Delayed Node | Removed in favour of `./witness_node --plugins "delayed_node"`. [Wiki](https://github.com/bitshares/bitshares-core/wiki/Delayed-Node) | Node | Removed |
```

Comment:
I would remove the line and place a link to "delayed node configuration" (or similar) in the `witness_node` entry above.